packages/app/src/embed/components/App/elements.js

```diff
@@ -33,7 +33,7 @@ export const Moving = styled.div`
   position: absolute;
   top: 0;
   bottom: 0;
-  left: props.sidebarOpen ? 250 : 0;
+  left: ${props => (props.sidebarOpen ? 250 : 0)}px;
```

Comment:
how did this not break completely until now 😱

Reply:
It did 😂